_data/governments.yml

```diff
@@ -1051,6 +1051,7 @@ United Nations:
   - UNStats
   - worldbank
   - WorldBank-Transport
+  - GFDRR
```

Comment:
This will need to be at the top of the list so the Alphabetical check passes on the CI